contrib/test_mergeBenchmarkSets/test_mergeBenchmarkSets.py

```diff
@@ -0,0 +1,538 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
```

Comment:
I guess this file could be `__init__.py`? Having `test_mergeBenchmarkSets.test_mergeBenchmarkSets.TestMergeBenchmarkSets.test_*` feels quite redundant.

Reply:
Renamed it.